suite-common/message-system/src/experimentUtils.ts

```diff
@@ -20,9 +20,8 @@ type ExperimentGetGroupByInclusion = {
  */
 export const getInclusionFromInstanceId = (instanceId: string, experimentId: ExperimentId) => {
     const combinedId = `${instanceId}-${experimentId}`;
-    const hash = createHash('sha256').update(combinedId).digest('hex').slice(0, 8);
 
-    return parseInt(hash, 16) % 100;
+    return getIntegerInRangeFromString(combinedId, 100);
```

Comment:
@matejkriz do you know if there is an issue if we change this and some users that before were falling into group A now they will follow into group B because the combination of `instanceId` and `experimentalId` will produce different number with this change.

We are not sure if this is an issue.

Reply:
There hasn’t been any experiment in production yet, so it’s fine. 👌